_blog/2024-05-12-modernizing the code structure.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Modernizing the AntennaPod Code Structure"
+excerpt: How we spent the last 3 years restructuring the AntennaPod code base.
+date: "2024-05-12 22:00:00"
+image: "2024/codeStructureTeaser.jpg"
+author: bytehamster
+layout: blog
+guid: 997f9895-7723-4ed0-88ee-8077878100d0
+---
+
+{: .lead}
```

Comment:
I'm not a huge fan of the design of that lead paragraph. Just using the different font size looks like something went wrong, not like a headline. Anyway, that's something for a different PR.